File: 15-acronym-builder/acronym-builder.js
// Write code to create a function that accepts a string and returns an acronym for the given string

const acronymBuilder=(str)=>{

    const strArry=str.split(' ')
    const acronymResult=[]

    for(let i=0; i<strArry.length; i++){
        const firstLetter=strArry[i].charAt(0)
        acronymResult.push(firstLetter.toUpperCase())
    }

    return acronymResult.join('')
}

console.log(acronymBuilder('i love programming'))

File: 15-acronym-builder/acronym_builder.py
def acronym_builder(string):

    stringList=string.split(' ')
    acronymResult=[]

    for word in stringList:
        firstLetter=word[0]
        acronymResult.append(firstLetter.upper())

    return ''.join(acronymResult)

print(acronym_builder('i love programming'))
